### ISSUE TITLE
Pre-trained network for tao doesn't provide any tracking ids or labels

### ISSUE THREAD

**Issue by iAmJuan550** [SysCV/qdtrack#65]
```
for i, data in enumerate(data_loader):
    with torch.no_grad():
        result = model(return_loss=False, rescale=True, **data)
        trks = result['track_result']
```
The trks above has two keys: bbox_result and track result

When using the bdd100k, the bbox_result key provides a dictionary where each key is the track id and the value for that track id is another dictionary with two keys (bbox and label).

However, when I use the tao pretrained network, the bbox_result and track result simply give me lists. So, bbox_result now gives a list of numpy arrays where each numpy array has size (n, 6). How do I infer the track id and label from each of these numpy arrays? The first value of each row of the numpy array seems like an integer but it has a really high value that keeps going up and down, so I guess it is the label? I don't see any other integer value in this numpy array

**Comment by OceanPang** [comment]
Hi @iAmJuan550, we have updated the docs and codes for TAO. Please pull the latest code and refer to [here](configs/tao/README.md).

**Comment by iAmJuan550** [comment]
Thank you, your updated instructions fixed my issue. For anyone wondering how to use the pretrained tao neural net, follow the step by step instructions [here](https://github.com/SysCV/qdtrack/issues/61#issuecomment-896934262) and then make the following changes in `custom_qdtrack.py`

```
cfg = Config.fromfile('configs/tao/ft_qdtrack_frcnn_r101_fpn_24e_tao.py')
...
...
checkpoint = load_checkpoint(model, 'weights/qdtrack_tao_20210812_221438-b6bd07e2.pth', map_location='cuda')

for i, data in enumerate(data_loader):
    with torch.no_grad():
        result = model(return_loss=False, rescale=True, **data)
        trks = result['track_result']
        if isinstance(trks, list):
            tao_dict = {}
            for trk in trks:
                if trk.size == 0:
                    continue
                for _trk in trk:
                    tao_dict[int(_trk[0])] = {'bbox': _trk[1:], 'label': int(_trk[0])}
            trks = tao_dict
        if len(trks) == 0:
            continue
```
